src/components/ui/FfaTabbedListingsComponent.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="columns-container">
+    <div class="column is-3 tabs-container">
+      <button 
```

Comment:
Whether or not bulma is the correct css framework is still open for discussion but **not** using the installed css framework is never right.  The first standing order is always: _Never roll your own_.